### PR TITLE
Fixed the parsing of parts in a ListParts xml record

### PR DESCRIPTION
**Pull request by etorreborre**
by reverting to the amazonka code in previous versions.

With this fix I can implement ambiata/mismi/#108.

### CODE REVIEW

**Comment by nhibberd** [comment]
Looks good.
On a side note, we almost need to keep a manifest of changes we are adding here and a reference if they have been raised/fixed on the official repository

**Comment by etorreborre** [comment]
Amazonka issue raised [here](https://github.com/brendanhay/amazonka/issues/122).

File: CHANGES.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Changes
+
+List of changes on this fork:
+
+ * `99.0.0`: fixed the `CopyObject` query (don't send a body on PUT and fix the XML parsing of `CopyObjectResult`)
```
Comment (markhibberd) [inline CHANGES.md]: @charleso @nhibberd Were there more before this? For example we regen'd everything to fix the missing instance state enums at some point as well.

Comment (markhibberd) [inline CHANGES.md]: Also should put amazonka issues and fix version here if we can, so we can rebase / clean-up soon.

Comment (nhibberd) [inline CHANGES.md]: As far as S3 that is it, not sure about SQS

Comment (etorreborre) [inline CHANGES.md]: Yes I forgot to ask. I tried to retro-fill from the git log but I don't know about the full history.

Comment (markhibberd) [inline CHANGES.md]: This is a good start anyway, just checking if there are others.

Comment (nhibberd) [inline CHANGES.md]: 99.0.1 has been fixed on this work branch, hasn't yet reached `develop`

**Pull request merged**